src/org/mediawiki/MediaWikiPhpPredefinedCodeStyle.java

```diff
@@ -42,7 +42,8 @@ public void apply(CodeStyleSettings settings) {
         commonSettings.SPACE_WITHIN_CATCH_PARENTHESES = true;
         commonSettings.SPACE_WITHIN_SWITCH_PARENTHESES = true;
         commonSettings.SPACE_WITHIN_ARRAY_INITIALIZER_BRACES = true;
-        commonSettings.SPACE_AFTER_TYPE_CAST = true;
+        commonSettings.SPACE_AFTER_TYPE_CAST = false;
+        commonSettings.SPACES_AROUND_VAR_WITHIN_BRACKETS = false;
```

Comment:
Looks like this one actually upsets it at compile... Commeting it out for now in my rebuild

Reply:
fixed by https://github.com/reedy/phpstorm-plugin-mediawiki/commit/9c45c6f15e6b7dd53d12858d70a8163da218a807, I think...